curvallis/run.py

```diff
@@ -417,41 +422,41 @@ def key_press_callback(self, event):
                     else:
                         self._selector.set_active(False)
                         print ("Block Select Disabled")
-            elif event.key == 'Q':
-                self._get_equation()
-            elif event.key == 'u':
-                #Undo the last point manipulation
+            elif event.key == 'Q':              #If 'Q' pressed
+                self._get_equation()            #Get equation from user to plot
+            elif event.key == 'u':              #If 'u' pressed
+                                                #Undo the last point manipulation
                 self._regions.undo()
                 self._canvas.draw()
-            elif event.key == 'Z':
-                if self._move_set == True:
+            elif event.key == 'Z':              #If 'Z' pressed
+                if self._move_set == True:      #Run TriLocal Smoothing
                     self._regions.smooth_data("trilocal", self._xmin, self._xmax, 
                                               self._ymin, self._ymax)
                     self._canvas.draw()
                 else:
                     print ("Select a region to smooth by pressing 'e'.")
-            elif event.key == 'X':
-                if self._move_set == True:
+            elif event.key == 'X':              #If 'X' pressed
+                if self._move_set == True:      #Run Integral Smoothing
                     self._regions.smooth_data("integral", self._xmin, self._xmax, 
                                               self._ymin, self._ymax)
                     self._canvas.draw()
                 else:
                     print ("Select a region to smooth by pressing 'e'.")
-            elif event.key == 'C':
-                if self._move_set == True:
+            elif event.key == 'C':              #If 'C' pressed         (not in key mapping)
+                if self._move_set == True:      #Run TriIntegral Smoothing (No longer exists)
                     self._regions.smooth_data("triintegral", self._xmin, self._xmax, 
                                               self._ymin, self._ymax)
                     self._canvas.draw()
                 else:
                     print ("Select a region to smooth by pressing 'e'.")
```

Comment:
As @sudo-Eric mentions, if this smoothing function no longer exists and the key mapping isn't present, should we just delete this whole block of code @leek2?

Reply:
What the heck, I swear I responded to this.
Anyway, yeah, if triintegral smoothing isn't included, go ahead and delete all that code.
I do wonder why it isn't there though.  It wasn't that great, so it's not really a problem, but it is curious.  Did Paul just never finish including it, or what?